src/sqlite3gen.cpp

```diff
@@ -893,18 +1558,102 @@ static void generateSqlite3ForMember(const MemberDef *md, const Definition *def)
   // enum values are written as part of the enum
   if (md->memberType()==MemberType_EnumValue) return;
   if (md->isHidden()) return;
-  //if (md->name().at(0)=='@') return; // anonymous member
 
-  // group members are only visible in their group
-  //if (def->definitionType()!=Definition::TypeGroup && md->getGroupDef()) return;
```

Comment:
These conditionals were already commented out in the XML they got copied in from, so I cut them here.